packages/arb-token-bridge-ui/src/components/common/HeaderNetworkInformation.tsx

```diff
@@ -16,25 +16,13 @@ export function HeaderNetworkInformation() {
     [l1Network, l2Network, isConnectedToArbitrum]
   )
 
-  const logoSrc = useMemo(() => {
-    const { isArbitrum, isArbitrumNova } = isNetwork(network.chainID)
-
-    if (!isArbitrum) {
-      return '/EthereumLogo.webp'
-    }
-
-    if (isArbitrumNova) {
-      return '/ArbitrumNovaLogo.webp'
-    }
-
-    return '/ArbitrumOneLogo.svg'
-  }, [network])
+  const logoSrc = useMemo(() => getNetworkLogo(network.chainID), [network])
```

Comment:
`getNetworkLogo` is a simple switch, I don't think we need to use memo here

Reply:
Done